yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/filters/DocumentRestrictionsFilterBuilder.java

```diff
@@ -0,0 +1,17 @@
+package com.yoti.api.client.docs.session.create.filters;
+
+import java.util.List;
+
+public interface DocumentRestrictionsFilterBuilder {
+
+    DocumentRestrictionsFilterBuilder forWhitelist();
+
+    DocumentRestrictionsFilterBuilder forBlacklist();
+
+    DocumentRestrictionsFilterBuilder withDocumentRestriction(List<String> countryCodes, List<String> documentTypes);
```

Comment:
I can't really think of a fitting name for the `withDocumentRestriction(List<String> countryCodes, List<String> documentTypes);` method. 
So unless we can think of a good name, I'm happy to just have one of these for the languages where method overloading isn't supported.

Do we think the 1st one would be preferable? Since the object is quite simple, and building it doesn't gain much for us?

Reply:
I think `withDocumentRestriction(DocumentRestriction documentRestriction)` is preferable - this is what I've changed it to in PHP/Node - I can re-revert this?

Reply:
Discussed offline and agreed overloading in languages that can makes sense, but use `withDocumentRestriction(DocumentRestriction documentRestriction)` as the preferred method